### ISSUE TITLE
Last commit inserted third party script

### ISSUE THREAD

**Issue by chemedev** [vtex-apps/react-app-template#71]
In last [commit](https://github.com/vtex-apps/react-app-template/commit/30b389bb40b2da76ba40e39aa66a10ec10d88896) someone intentionally added metatags with facebook domain verification, this should be removed ASAP since it could be used by any developer and kept in code without notice.

**Comment by chemedev** [comment]
Yes, and it looks like nobody is taking care of issues in this repository... 🤦🏽

**Comment by Mecrano** [comment]
Solved, thanks for reporting
